lib/postgrex/query.ex

```diff
@@ -22,10 +23,13 @@ defmodule Postgrex.Query do
     result_formats: [:binary | :text] | nil,
     decoders:       [Postgrex.Types.oid] | [(binary -> term)] | nil,
     types:          Postgrex.TypeServer.table | nil,
-    null:           atom}
+    null:           atom,
+    max_rows:       integer,
+    state:          :suspended | :done | nil
```

Comment:
A query prepared on one connection should work on another connection to the same database.